package.json

```diff
@@ -38,17 +38,18 @@
     "test": "jest"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
+    "@babel/cli": "^7.0.0-rc.3",
+    "@babel/core": "^7.0.0-rc.3",
+    "@babel/plugin-syntax-jsx": "^7.0.0-rc.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0-rc.3",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^8.2.6",
     "classnames": "^2.2.5",
     "eslint": "^5.4.0",
     "eslint-config-prettier": "^3.0.1",
     "glob": "^7.1.2",
     "husky": "^0.14.3",
-    "jest": "^23.5.0",
```

Comment:
jest 23 is not compatible with babel 7: 
https://github.com/facebook/jest/pull/4923

Reply:
issue: https://github.com/facebook/jest/issues/6913